src/Symfony/Component/Config/Builder/ConfigBuilderGenerator.php

```diff
@@ -60,7 +60,7 @@ public function NAME(): string
 {
     return \'ALIAS\';
 }
-        ', ['ALIAS' => $rootNode->getPath()]);
+', ['ALIAS' => $rootNode->getPath()]);
```

Comment:
Lets end the line on }. 

Ie

```php
}', ['ALIAS' => $rootNode->getPath()]);
```

Same on the two other places below.

Reply:
@Nyholm Done